### ISSUE TITLE
Bug: not showing WebP correctly

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [McoyJiang/Android-WebP#7]
Try this attached animated WebP file.

[test.zip](https://github.com/McoyJiang/Android-WebP/files/2521983/test.zip)

This is what I get:

![image](https://user-images.githubusercontent.com/5357526/47606530-474dfc00-da1d-11e8-952a-aeafb0a96f9e.png)

While in fact it should look like this:

![image](https://user-images.githubusercontent.com/5357526/47606538-53d25480-da1d-11e8-8563-65aac479aacb.png)

**Comment by peerless2012** [comment]
I use FrameSequence and have the same issue.

So it may be is FrameSequence problem。

**Comment by AndroidDeveloperLB** [comment]
@peerless2012 This issue doesn't exist on other places.
Do you know of a good alternative library for animated GIF and WEBP ?
This one is pretty old.

**Comment by peerless2012** [comment]
Yes, I fix this issue in my project. You should process the blend current result with pre result.
